android/test-app-util.gradle

```diff
@@ -145,3 +145,42 @@ ext.getReactNativeVersionNumber = { baseDir ->
     def (major, minor, patch) = manifest["version"].findAll(/\d+/)
     return (major as int) * 10000 + (minor as int) * 100 + (patch as int)
 }
+
+ext.getSigningConfigs = { baseDir ->
+    def safeSetMap = { varName, map, prop, defaultVal ->
+        map[varName] = prop.containsKey(varName) ? prop.get(varName) : defaultVal
+    }
+
+    def definedConfigs = new LinkedHashMap<String, Object>()
+    def manifestFile = findFile("app.json")
+    if (manifestFile != null) {
+        def manifest = new JsonSlurper().parseText(manifestFile.text)
+
+        if (!manifest["android"]) {
+            return definedConfigs
+        }
+
+        def signingConfigs = manifest["android"]["signingConfigs"]
+        if (signingConfigs) {
+            signingConfigs.each { config ->
+                def configName = config.key
+                def props = config.value
+                def pathStoreFile = props.containsKey("storeFile")
+                    ? Paths.get(rootDir.toString(), props.get("storeFile")).normalize().toAbsolutePath()
```

Comment:
Should this be relative to `app.json`? `rootDir` depends on where the `build.gradle` file lives, iirc.

Reply:
Great find. Yes, you are correct. Pushed a change for that to use the manifest's location as a reference.